sky/utils/kubernetes_enums.py

```diff
@@ -27,3 +27,24 @@ class KubernetesServiceType(enum.Enum):
     """Enum for the different types of services."""
     NODEPORT = 'NodePort'
     CLUSTERIP = 'ClusterIP'
+
+
+class KubernetesPortMode(enum.Enum):
+    """Enum for the different types of modes supported for opening
+    ports on Kubernetes.
+    """
+    INGRESS = 'ingress'
+    LOADBALANCER = 'loadbalancer'
```

Comment:
```suggestion
    LOAD_BALANCER = 'loadbalancer'
```
nit

Reply:
I think it's best to keep `LOADBALANCER` since that's how the object is named in [Kubernetes](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer)